slices/util-linux.yaml

```
@@ -0,0 +1,271 @@
+      - libcap-ng0_libs
+      - libmount1_libs
+      - libpcre2-8-0_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libudev1_libs
+      - libuuid1_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/findmnt:
+      /usr/bin/mountpoint:
+      /usr/sbin/findfs:
+      /usr/sbin/fsck:
+      /usr/sbin/fsck.cramfs:
+      /usr/sbin/fsck.minix:
+      /usr/sbin/fsfreeze:
+      /usr/sbin/fstrim:
+      /usr/sbin/isosize:
+      /usr/sbin/mkfs:
+      /usr/sbin/mkfs.bfs:
+      /usr/sbin/mkfs.cramfs:
+      /usr/sbin/mkfs.minix:
+      /usr/sbin/mkswap:
+      /usr/sbin/swaplabel:
+      /usr/sbin/switch_root:
+      /usr/sbin/wipefs:
+
+  # The following emulates the post-install script.
+  generated:
+    essential:
+      # Since the more binary is located at ./usr/bin/more and /bin in noble
+      # points to /usr/bin via base-files, the base-files_bin slice is included
+      # here.
+      - base-files_bin
+      - util-linux_file
+    contents:
+      /usr/bin/pager: {symlink: /bin/more}
+
+  setarch:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/bin/i386: { arch: [i386, amd64] }
+      /usr/bin/linux32:
+      /usr/bin/linux64:
+      /usr/bin/ppc: { arch: [ppc64el] }
+      /usr/bin/ppc32: { arch: [ppc64el] }
+      /usr/bin/ppc64: { arch: [ppc64el] }
+      /usr/bin/s390: { arch: [s390x] }
+      /usr/bin/s390x: { arch: [s390x] }
+      /usr/bin/setarch:
+      /usr/bin/x86_64: { arch: [amd64] }
+
+  proc:
+    essential:
+      - libc6_libs
+      - libcap-ng0_libs
+      - libsmartcols1_libs
+    contents:
+      /usr/bin/choom:
+      /usr/bin/chrt:
+      /usr/bin/ionice:
+      /usr/bin/prlimit:
+      /usr/bin/setpriv:
+      /usr/bin/setsid:
+      /usr/bin/taskset:
+      /usr/bin/uclampset:
+
+  text-dev:
+    essential:
+      - libc6_libs
+      - libtinfo6_libs
+    contents:
+      /usr/bin/setterm:
+      /usr/sbin/agetty:
+      /usr/sbin/getty:
+      /usr/sbin/ldattach:
+
+  namespace:
+    essential:
+      - libblkid1_libs
+      - libc6_libs
+      - libmount1_libs
+      - libpcre2-8-0_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+    contents:
+      /usr/bin/lsns:
+      /usr/bin/nsenter:
+      /usr/bin/unshare:
+
+  kernel:
+    essential:
+      - libc6_libs
+      - libtinfo6_libs
+    contents:
+      /usr/bin/dmesg:
+      /usr/sbin/readprofile:
+
+  login:
+    essential:
+      - libc6_libs
+      - libcap-ng0_libs
+      - libcrypt1_libs
+      - libgcrypt20_libs
+      - libgpg-error0_libs
+      - liblz4-1_libs
+      - liblzma5_libs
+      - libpcre2-8-0_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libsystemd0_libs
+      - libzstd1_libs
+    contents:
+      /usr/bin/last:
+      /usr/bin/lastb:
+      /usr/bin/lslogins:
+      /usr/bin/utmpdump:
+      /usr/sbin/sulogin:
+
+  timers:
+    essential:
+      - libc6_libs
+      - libsmartcols1_libs
+    contents:
+      /usr/bin/wdctl:
+      /usr/sbin/rtcwake:
+
+  shell:
```

```suggestion
  shell-command:
```

Or, something else that indicates that these are commands which are useful to run in shell.

---

Renamed as `cli-helpers`, since `shell-command` is too general. Also named with plural form as these helpers are heterogeneous in their functionalities.